docs/reference/troubleshooting/common-issues/disk-usage-exceeded.asciidoc

```diff
@@ -102,19 +102,34 @@ PUT _cluster/settings
 [[fix-watermark-errors-resolve]]
 ==== Resolve
 
-As a long-term solution, we recommend you do one of the following best suited 
+To resolve, we recommend you do one of the following best suited 
 to your use case: 
 
-* add nodes to the affected <<data-tiers,data tiers>>
-+
-TIP: You should enable <<xpack-autoscaling,autoscaling>> for clusters deployed using our {ess}, {ece}, and {eck} platforms.
+* horizontally scale nodes of the affected <<data-tiers,data tiers>>
 
-* upgrade existing nodes to increase disk space
-+
-TIP: On {ess}, https://support.elastic.co[Elastic Support] intervention may 
-become necessary if <<cluster-health,cluster health>> reaches `status:red`. 
+* vertically scale existing nodes to increase disk space
 
-* delete unneeded indices using the <<indices-delete-index,delete index API>>
+* delete indices using the <<indices-delete-index,delete index API>>, either
+permanently if unneeded or temporarily to later 
+<<snapshots-restore-snapshot,snapshot restore>>
 
 * update related <<index-lifecycle-management,ILM policy>> to push indices 
 through to later <<data-tiers,data tiers>>
+
+TIP: On {ess} and {ece}, indices may need to be temporarily deleted via
+its {cloud}/ec-api-console.html[Elasticsearch API Console] to later
+<<snapshots-restore-snapshot,snapshot restore>> in order to resolve
+<<cluster-health,cluster health>> `status:red` which will block
+{cloud}/ec-activity-page.html[attempted changes]. If you experience issues
+with this resolution flow on {ess}, kindly reach out to
+https://support.elastic.co[Elastic Support] for assistance.
```

Comment:
can we explain why they have to do the deletion here? how does someone determine if this note applies to them?